### PR TITLE
feat: handle connector's signals in nim

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Support PR for https://github.com/status-im/status-desktop/pull/15565

Add handling code for connector's signals

### CODE REVIEW

File: src/app/core/signals/remote_signals/connector.nim [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  dAppUrl*: string
+  chainID*: int
+  txArgsJson: string
+
+proc fromEvent*(T: type ConnectorSendRequestAccountsSignal, event: JsonNode): ConnectorSendRequestAccountsSignal =
+  result = ConnectorSendRequestAccountsSignal()
+  result.dAppUrl = event["event"]{"dAppUrl"}.getStr()
+  result.dAppName = event["event"]{"dAppName"}.getStr()
+  result.dAppIconUrl = event["event"]{"dAppIconUrl"}.getStr()
+  echo "--------> ConnectorSendRequestAccountsSignal: ", result.dAppUrl, " ", result.dAppName, " ", result.dAppIconUrl
```
Comment (jrainville) [inline src/app/core/signals/remote_signals/connector.nim:21]: leftover? Could be cnnverted to info I guess

File: src/app/core/signals/remote_signals/connector.nim [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+proc fromEvent*(T: type ConnectorSendTransactionSignal, event: JsonNode): ConnectorSendTransactionSignal =
+  result = ConnectorSendTransactionSignal()
+  result.dAppUrl = event["event"]{"dAppUrl"}.getStr()
+  result.chainID = event["event"]{"chainID"}.getInt()
+  result.txArgsJson = event["event"]{"txArgsJson"}.getStr()
+  echo "--------> ConnectorSendTransactionSignal: ", result.dAppUrl, " ", result.chainID, " ", result.txArgsJson
```
Comment (jrainville) [inline src/app/core/signals/remote_signals/connector.nim:28]: Same here

**Review by jrainville — APPROVED** [review]
LGTM

**Comment by kounkou** [comment]
Do we want to create a higher level Signal, something like `Connector` similar to `Wallet` then have sub signals for `ConnectorSendRequestAccounts` and `ConnectorSendTransactionSignal`.
In summary adding a Type ?
Other that this comment, it works pretty well to receive the signal from the connector service on nim side 👍🏾

File: src/app/core/signals/remote_signals/connector.nim (hunk 1 above)
Comment (MishkaRogachev) [reply src/app/core/signals/remote_signals/connector.nim:21]: Apologies, i forgot to draft this PR, it's not for merging

**Pull request closed without merging**